src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -64,8 +65,14 @@ public abstract class VariablesStorage implements Closeable {
 	
 	// created in the constructor, started in load()
 	private final Thread writeThread;
-	
-	protected VariablesStorage(final String name) {
+
+	/**
+	 * Construct a variable storage object.
+	 * 
+	 * @param name The configuration name that was used. This will be inserted by Skript.
+	 */
+	protected VariablesStorage(@NonNull String name) {
```

Comment:
The NonNull annotation shouldn't be needed, as all parameters are NonNull by default 

This may not be the case for this package, so please feel free to correct me if I am wrong :)